libc/src/__support/OSUtil/linux/fcntl.cpp

```diff
@@ -86,17 +86,37 @@ int fcntl(int fd, int cmd, void *arg) {
     libc_errno = -ret;
     return -1;
   }
+  case F_GETLK: {
```

Comment:
Look like we can just put all these 3 cases inside `#ifdef SYS_fcntl64` and remove the `default` case.